### PR TITLE
Update max_bytes_for_level_base unit to bytes and remove unuseful error checker.

### PR DESCRIPTION
**Pull request by WyattJia**
This refers to #612

### CODE REVIEW

File: src/config.cc [hunk 1]
```diff
@@ -449,26 +449,17 @@ void Config::initFieldCallback() {
       }},
       {"rocksdb.max_bytes_for_level_base", [this](Server* srv, const std::string &k, const std::string& v)->Status {
         if (!srv) return Status::OK();
-        if (!RocksDB.level_compaction_dynamic_level_bytes) {
```
Comment (tisonkun) [inline src/config.cc:452]: Copy comment from https://github.com/apache/incubator-kvrocks/pull/611#discussion_r884190774:

Why do you make these changes?

Comment (tisonkun) [reply src/config.cc:452]: It seems another option and with this patch you remove all usages of this config. There must be a motivation and if you remove all usages, perhaps it's necessary to remove the option also.

Comment (WyattJia) [reply src/config.cc:452]: > Copy comment from [#611 (comment)](https://github.com/apache/incubator-kvrocks/pull/611#discussion_r884190774):
> 
> Why do you make these changes?

After my discussion with @ShooterIT , we believe that this configuration item may not be required. If the configuration is not filled in, an error should not be returned, and the default value should be set to false.

**Comment by ShooterIT** [comment]
@WyattJia please also check again
```
  options->max_bytes_for_level_base = config_->RocksDB.max_bytes_for_level_base * MiB;
```
need to remove `* MiB`

**Comment by WyattJia** [comment]
> @WyattJia please also check again
> 
> ```
>   options->max_bytes_for_level_base = config_->RocksDB.max_bytes_for_level_base * MiB;
> ```
> 
> need to remove `* MiB`

@ShooterIT @tisonkun I have updated changes about unit and error config checker.Please check and review. Very thankful.

**Review by ShooterIT — APPROVED** [review]

**Review by git-hulk — APPROVED** [review]

**Comment by git-hulk** [comment]
Thanks @WyattJia, merged.

**Pull request merged**